pkgs/data/documentation/man-pages/default.nix

```diff
@@ -1,15 +1,34 @@
-{ lib, stdenv, fetchurl }:
+{ lib, stdenv, groff, fetchurl }:
 
 stdenv.mkDerivation rec {
   pname = "man-pages";
-  version = "5.13";
+  version = "6.05";
 
   src = fetchurl {
     url = "mirror://kernel/linux/docs/man-pages/${pname}-${version}.tar.xz";
-    sha256 = "sha256-YU2uPv59/UgJhnY6KiqBeSFQMqWkUmwL5eiZol8Ja4s=";
+    sha256 = "sha256-ibFEXP4uPei9E5dYx48Is3gTz/IXufscjfVf2UB4daY=";
   };
 
+  buildInputs = [
+    groff
+  ];
+
   makeFlags = [ "prefix=$(out)" ];
+
+  # Only build regular man pages
+  # Don't build html or pdf for now
+  buildFlags = [ "build-catman" ];
+
+  # Append `.*` to share/lint/groff/man.ignore.grep to ignore troff errors
+  preBuild = ''
+    # troff warnings cause the build to fail
+    # (see the `groff_man_ignore_grep` make variable and
+    # the `_CATMAN_MAN_set` build target in share/mk/build/catman.mk)
+
+    # Append .* to man.ignore.grep to effectively ignore groff errors
```

Comment:
It's an antipattern to have comments in indented strings like this if avoidable, because changing them triggers a rebuild (which isn't the case if they're outside of it).